common/flatpak-run.c

```diff
@@ -609,6 +609,46 @@ flatpak_run_add_cups_args (FlatpakBwrap *bwrap)
                           NULL);
 }
 
+static void
+flatpak_run_add_gpg_agent_args (FlatpakBwrap *bwrap)
+{
+  const char * agent_socket;
+  g_autofree char * sandbox_agent_socket = NULL;
+  g_autoptr(GError) gpgconf_error = NULL;
+  g_autoptr(GSubprocess) process = NULL;
+  g_autoptr(GInputStream) base_stream = NULL;
+  g_autoptr(GDataInputStream) data_stream = NULL;
+
+  process = g_subprocess_new (G_SUBPROCESS_FLAGS_STDOUT_PIPE,
+                    &gpgconf_error,
+                    "gpgconf", "--list-dir", "agent-extra-socket", NULL);
```

Comment:
Why are we mounting extra socket to the normal socket path (i.e. `S.gpg-agent.extra` to `S.gpg-agent`)?

Reply:
Maybe at least add a comment on why this is done this way?

Reply:
Check my updated comment above. Since it seems like a non-main socket, I replaced it according to your suggestion.